.github/workflows/conformance-aks-v1.10.yaml

```diff
@@ -214,7 +214,7 @@ jobs:
         shell: bash
         run: |
           for image in cilium-ci operator-azure-ci hubble-relay-ci ; do
-            until docker manifest inspect quay.io/${{ github.repository_owner }}/$image:${{ steps.vars.outputs.sha }} &> /dev/null; do sleep 45s; done
+            until docker manifest inspect quay.io/${{ secrets.QUAY_ORGANIZATION }}/$image:${{ steps.vars.outputs.sha }} &> /dev/null; do sleep 45s; done
```

Comment:
The other lines in this file did `QUAY_ORGANIZATION_DEV`. Is there a reason this one didn't?

Reply:
(This seems to be a repeated problem with this pattern, so I won't comment on the others)